backend/static/scripts/pong.js

```diff
@@ -1247,7 +1247,8 @@ function setupDropdownListeners() {
 
 function visual() {
     if (game.isGameRunning() == true) {
-        var myModal = new bootstrap.Modal('#visualmodal');
+        game.pause();
```

Comment:
When imp mode is pressed during game I understand that we do not need to pause the game, only display a message indicating that this is not allowed. If you check the main branch you will notice the ball is not stopping.
We don't need this line. 1250

Reply:
isnt that problematic? the game keeps proceeding and then it can even end

Reply:
If that is the case, what we need to do is include the resume similarly to what we did for the configuration (applyConfiguration function). If you prefer to follow this approach I can pause/resume accordinly.